tools/perf/configs/dashboard/smoke-chrome.json5

```diff
@@ -37,6 +37,13 @@
         "load:ntp:brave_news:2024",
       ]
     },
+    {
+      "name": "system_health.memory_desktop",
```

Comment:
nit: better to have a single block with `"name ": "system_health.memory_desktop"` because some code can assume that the benchmarks are unique.

Reply:
Okay, Will move it to the existing block.